src/etc/inc/interfaces.inc

```diff
@@ -121,8 +121,15 @@ function does_vip_exist($vip) {
 	}
 
 	$ifacedata = pfSense_getall_interface_addresses($realif);
+
+	if (is_ipaddrv6("{$vip['subnet']}/{$vip['subnet_bits']}")) {	
```

Comment:
`is_ipaddrv6()` will always return false if you pass it a value with a prefix length like that. Maybe you want `is_subnetv6()`?

Reply:
ok, now I know
fixed